x/ibc/04-channel/keeper/timeout_test.go

```diff
@@ -136,34 +133,26 @@ func (suite *KeeperTestSuite) TestTimeoutOnClose() {
 	var (
 		packet      types.Packet
 		nextSeqRecv uint64
-		proofHeight uint64            = 3
+		proofHeight uint64            = 1
 		proof       commitment.ProofI = ibctypes.InvalidProof{}
 		proofClosed commitment.ProofI = ibctypes.InvalidProof{}
 	)
 
 	testCases := []testCase{
-		// {"success", func() {
-		// 	nextSeqRecv = 1
-		// 	proofHeight = 100
-		// 	packet = types.NewPacket(newMockTimeoutPacket(3), 2, testPort1, testChannel1, counterparty.GetPortID(), counterparty.GetChannelID())
-		// 	suite.createClient(testClientID1)
-		// 	suite.createClient(testClientID2)
-		// 	suite.createConnection(testConnectionID1, testConnectionID2, testClientID1, testClientID2, connectionexported.OPEN)
-		// 	suite.createChannel(testPort1, testChannel1, testPort2, testChannel2, exported.OPEN, exported.ORDERED, testConnectionID1)
-		// 	suite.createChannel(testPort2, testChannel2, testPort1, testChannel1, exported.OPEN, exported.ORDERED, testConnectionID1)
-		// 	suite.app.IBCKeeper.ChannelKeeper.SetPacketCommitment(suite.ctx, testPort1, testChannel1, 2, types.CommitPacket(packet.Data))
-		// 	suite.app.IBCKeeper.ChannelKeeper.SetNextSequenceRecv(suite.ctx, testPort1, testChannel1, nextSeqRecv)
-		// }, true},
+		{"success", func() {
+			packet = types.NewPacket(mockSuccessPacket{}, 2, testPort1, testChannel1, counterparty.GetPortID(), counterparty.GetChannelID())
+			suite.createClient(testClientID1)
+			suite.createConnection(testConnectionID1, testConnectionID2, testClientID1, testClientID2, connectionexported.OPEN)
+			suite.createChannel(testPort1, testChannel1, testPort2, testChannel2, exported.OPEN, exported.UNORDERED, testConnectionID1)
+			suite.app.IBCKeeper.ChannelKeeper.SetPacketCommitment(suite.ctx, testPort1, testChannel1, 2, types.CommitPacket(packet.Data))
+			suite.app.IBCKeeper.ChannelKeeper.SetNextSequenceRecv(suite.ctx, testPort1, testChannel1, nextSeqRecv)
+		}, true},
 		{"channel not found", func() {}, false},
-		{"channel not open", func() {
```

Comment:
why was this test case removed?

Reply:
`TimeoutOnClose` doesn't check for the state of the channel